src/lib/components/Date/Date.scss

```diff
@@ -2,11 +2,17 @@
   display: flex;
   flex-direction: column;
   text-align: center;
-  width: 78px;
-  height: calc(39px * 2 + 2px);
```

Comment:
@Benaiah was there a reason for the additional `2px` here? I adjusted it below so that it renders as a square since `box-sizing` should be set to `border-box`.